### ISSUE TITLE
DataLoader

### ISSUE THREAD

**Issue by MetroTY** [chuangchuangtan/FreqNet-DeepfakeDetection#3]
Hi author, I would like to ask the training set, test set, and validation set are in the corresponding CNNDetection?

> https://github.com/PeterWang512/CNNDetection

Where the training set is using the four categories car,cat,chair,horse from CNNDetection right?Test set, validation set are both used right?
